Modules/Compatibility/README

```diff
@@ -6,4 +6,4 @@ considered as Legacy code, and will be removed from the toolkit
 in future releases.
 
 Deprecated:  Any code that provides a feature or function
```

Comment:
Unless a `.gitkeep` file is added to `Deprecated`, the folder will be removed, as this appears to remove all of its contents, if I'm not wrong. So not sure whether this comment would hold.

Reply:
There are some temporary files created as placeholders to ease future deprecations.